package.json

```diff
@@ -30,7 +30,10 @@
   },
   "peerDependencies": {
     "react": "^16.8.0 || ^17.0.0",
-    "react-router-dom": "^4.0.0 || ^5.0.0"
+    "react-redux": "^7.2.8",
+    "react-router-dom": "^4.0.0 || ^5.0.0",
+    "redux": "^4.1.2",
+    "redux-actions": "^2.6.5"
```

Comment:
These are all used in code that is **uncommented**. I used that as the proxy for "required".

Reply:
Makes sense!